page_creation_utility/README.md

```diff
@@ -0,0 +1,122 @@
+# Page Creation
+
+This project contains a [Java utility](https://github.com/HCL-TECH-SOFTWARE/dx-portlet-development-utilities/blob/main/pageimport/src/main/java/com/hcl/dx/sample/util/Utility.java) to create a single page using a default or by selecting an existing page template. The project gives an example source code that shows how to use the Controller and Model SPI calls related to WCM, for which the API calls will be handled inside of a JSP JSCR286 portlet that provides the graphical user interface.  
```

Comment:
Not sure what is meant with Java Utility? The project contains sample code to create pages.

Reply:
That was a suggestion by Herbert. Because our github repository has the name "dx-portlet-development-utilities", we want to show up or describe, why it is part of the repository. In functionality point of view it is a utility, because you can create Portal pages with that deployed project. But it is also some kind of utility, when you want to use that as an example code for your own coding.

Reply:
```suggestion
This project contains a [Java utility] to create a single page using a default template or by selecting an existing page template. The project gives example source code that shows how to use the Controller and Model SPI calls related to WCM, for which the API calls will be handled inside of a JSP JSR286 portlet that provides the graphical user interface.  
```